mathics/builtin/box/compilation.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from mathics.builtin.base import BoxConstruct
+from mathics.version import __version__  # noqa used in loading to check consistency.
```

Comment:
@mmatera @rocky do you think that checking consistency is still necessary. I think that Python wouldn't mess up with just some files. I think this would be either 0 or all.

Reply:
The only problem is to remove this line on each module...

Reply:
I'll do that.